server/auto_merge.go

```diff
@@ -74,9 +80,11 @@ func (s *Server) AutoMergePR() {
 			SHA:         prToMerge.Head.GetSHA(),
 			MergeMethod: "merge",
 		}
+
 		merged, _, err := client.PullRequests.Merge(context.Background(), pr.RepoOwner, pr.RepoName, pr.Number, "Automatic Merge", opt)
 		if err != nil {
 			errMsg := fmt.Sprintf("Error while trying to automerge the PR\nErr %s", err.Error())
+			s.removeLabel(pr.RepoOwner, pr.RepoName, pr.Number, s.Config.AutoPRMergeLabel)
```

Comment:
Should we also remove the label even if successful?

Reply:
yeah, makes sense, thanks!